Source/Details/_ASCollectionViewCell.mm

```diff
@@ -106,7 +106,8 @@ - (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event
    * superclass hitTest:withEvent: implementation. If this returns a valid value we can go on with
    * checking the node as it's expected to not be in one of these states.
    */
-  if (![super hitTest:self.bounds.origin withEvent:event]) {
+  CGPoint originPointOnView = [self convertPoint:self.node.view.bounds.origin fromView:self.node.view];
```

Comment:
Nit: For readability and a tiny tiny bit of perf, please store `self.node` and `self.node.view` in variables and use them throughout the method.